src/utils.ts

```diff
@@ -25,6 +25,13 @@ export class Utils {
         }
     }
 
+    public static getDeveloperEndpoint(backendUrl: string): string {
```

Comment:
Just in case, in @paperbits/common, there is a couple of functions that do similar things: https://github.com/paperbits/paperbits-common/blob/master/src/utils.ts#L367-L373